virtac/atip_ioc_entry.py

```diff
@@ -56,14 +60,39 @@ def main():
         not args.disable_emittance,
     )
 
-    # Add special case out record for SOFB to write to.
-    builder.SetDeviceName("CS-CS-MSTAT-01")
-    builder.aOut("FBHEART", initial_value=10)
+    # Warn if set to default EPICS port(s) as this will likely casue PV conflicts.
+    conflict_warning = ", this may lead to conflicting PV names on multiple servers."
```

Comment:
"this may lead to conflicting PV names with production IOCs"?

Reply:
changed